runtime/compiler/build/toolcfg/common.mk

```diff
@@ -29,6 +29,7 @@ PRODUCT_INCLUDES=\
     $(FIXED_SRCBASE)/compiler/$(TARGET_ARCH)/$(TARGET_SUBARCH) \
     $(FIXED_SRCBASE)/compiler/$(TARGET_ARCH) \
     $(FIXED_SRCBASE)/compiler \
+    $(FIXED_SRCBASE)/compiler/microjit \
```

Comment:
This should be conditional on `J9VM_OPT_MICROJIT`.

Reply:
It has been moved to an `J9VM_OPT_MICROJIT` guarded section.